lxd/instance.go

```diff
@@ -324,6 +324,8 @@ func instanceCreateAsCopy(s *state.State, opts instanceCreateAsCopyOpts, op *ope
 				// leave alone so we don't prevent copy.
 			}
 
+			srcSnap.LocalConfig()["volatile.uuid.generation"] = srcSnap.LocalConfig()["volatile.uuid"]
```

Comment:
This shouldn't be needed because the call to `instance.CreateInternal()` below should populate it to be the same as volatile.uuid.